Patica Andreea/FixMe.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+void print(int **a,int n,int m)
+{
+    int i,j;
+    for(i=0; i<n; i++)
+    {
+        for(j=0; j<m; j++)
+        {
+            printf(" %d ",a[i][j]);//
+        }
+        printf("\n");
+    }
+}
+
+void transpose(int **a,int n,int m)
+{
+    int i,j;
+    int** b=(int**)(malloc(sizeof(int)*m));
+    for(i=0; i<m; i++)
+        b[i]=(int*)(malloc(sizeof(int)*n));
+    for(i=0; i<n; i++)
+        for(j=0; j<m; j++)
+            b[j][i]=a[i][j];
+    print(b,m,n);
```

Comment:
I don't like this. You say the function `transposes` something, yet it doesn't.
It `PRINTS` the matrix transposition, but doesn't return it, doesn't change the matrix I've sent, it's very, very misleading.
You might not agree with me now, but you will soon enough. This function should either change the elements "in place" or return the transposed matrix.

Reply:
I did not write it!. This is the code in which you told us to correct the errors

Reply:
Woops